language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/CommandClient.java

```diff
@@ -17,7 +18,16 @@
 
 /** An RxJava version of {@link com.daml.ledger.api.v1.CommandServiceGrpc} */
 public interface CommandClient {
-
+  // TODO: Deprecate old methods and javadoc new ones
+  Single<Empty> submitAndWait(CommandClientConfig params);
+
+  /**
+   * This method has been deprecated as of [???]. Please use {@link
```

Comment:
Is there a `@since` annotation for this?

Reply:
Yes I could use `@since`. Need to confirm what version to use there though

Reply:
[`@since`](https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#since) isn't used for that in Javadoc. Use [the `@deprecated` Javadoc tag](https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#deprecated) instead; grep our repo for examples.